packages/odyssey-storybook/src/components/odyssey-labs/AppTile/AppTile.stories.tsx

```diff
@@ -317,3 +326,41 @@ export const CustomContent: StoryObj<AppTileProps> = {
     );
   },
 };
+
+export const Loading: StoryObj<AppTileProps> = {
+  render: function C() {
+    const [isDrawerOpen, setIsDrawerOpen] = useState<boolean>(false);
+
+    return (
+      <>
+        <Drawer
+          isOpen={isDrawerOpen}
+          onClose={() => setIsDrawerOpen(false)}
+          showDividers={false}
+        />
+        <Box sx={{ maxWidth: 262 }}>
+          <AppTile
+            actionAriaControls=""
+            actionAriaExpanded={isDrawerOpen}
+            actionAriaHasPopup="menu"
+            actionIcon={<SettingsIcon />}
+            actionLabel="Open app settings"
+            auxiliaryText="Single sign-on"
+            onActionClick={() => setIsDrawerOpen(true)}
+            onClick={() => alert("Open the app")}
```

Comment:
We don't need to add `alert` to Storybook. It tracks any `on`-prefixed functions using the built-in `action()` function. In this case, it would call `action("onClick")` and show up in the Actions area:
![image](https://github.com/user-attachments/assets/6b9dfe6c-a056-4b94-b201-bbd0503eaf74)